pkg/testdata/error-test-shard-identity-bulk.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
What's the error in this file?

Reply:
In this configured "assets": "test-value" is string but expected is array